wordpress-mobile/WordPress-iOS
# Display "ghost post card" when loading posts

Ref https://github.com/wordpress-mobile/WordPress-iOS/issues/11646

This PR shows a "ghost post card" when loading posts:

<img src="https://user-images.githubusercontent.com/7040243/58479337-7735bd00-812e-11e9-8b4b-f61c217a154d.png" width="300" />


To test:
- Check if the ghost cells appear when loading fresh content
- All the previous error and messages showed by `NoResultsController` should remain the same
- Check it in iPad and with RtL screens
